### PR TITLE
fixes README.md usage example

### PR DESCRIPTION
**Pull request by mikebrow**
Updates usage example.

Signed-off-by: Mike Brown brownwm@us.ibm.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,11 @@ Simple command line tool for fetching the [Open Container Initiative image forma
 ## Usage
 
 ```
-$ go get github.com/containers/oci-fetch
+$ go get github.com/containers/oci-fetch/oci-fetch
```
Comment (runcom) [inline README.md:8]: this should really be `github.com/containers/oci-fetch/cmd/oci-fetch` to follow go idioms

Comment (mikebrow) [inline README.md:8]: Don't disagree.. just making it work.  Can move it add a cmd prefix directory for oci-fetch with another PR if you like.

Comment (runcom) [inline README.md:8]: Yes, please

**Comment by runcom** [comment]
LGTM

**Pull request merged**